### DISCUSSION TITLE
NGROK tunnel

### DISCUSSION THREAD

**Discussion by neharaut00** [miguelgrinberg/flask-sock#72] [Q&A]
Hi! I followed Miguel's guide on Flask-Sock and everything worked perfectly. However, after tunneling the app to NGROK i am unable to see result on web page. 
![image](https://github.com/miguelgrinberg/flask-sock/assets/88868603/c6958fd9-3d03-4300-a966-f5b81cc1bdb2)
![image](https://github.com/miguelgrinberg/flask-sock/assets/88868603/2471a9fe-8678-4b79-b0cc-4b41216c9afc)

**Comment by miguelgrinberg** [comment]
What error do you get? The screenshot appears to show a working application.
